Biz/Model/t/MailPartList/8.in

```diff
@@ -0,0 +1,29 @@
+Return-Path: <nagler@bivio.biz>
+Received: from ice.bivio.biz (bkp.test.bivio.com [216.87.82.252])
+	by bivio.biz (8.11.6/8.11.6) with ESMTP id k0LIFCq13191
+	for <nagler@bivio.biz>; Sat, 21 Jan 2006 11:15:12 -0700
+Received: from dhcp72.n.bivio.com.bivio.biz (localhost.localdomain [127.0.0.1])
+	by ice.bivio.biz (8.11.6/8.11.6) with ESMTP id k0LIF5P24345;
+	Sat, 21 Jan 2006 11:15:06 -0700
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="0__=0ABBF8F6DFC246528f9e8a93df938690918c0ABBF8F6DFC24652"
+Content-Transfer-Encoding: 7bit
+Message-ID: <17362.31273.581812.210650@dhcp72.n.bivio.com>
+X-Mailer: VM 7.19 under Emacs 22.0.50.1
+Status:  O
+From: Rob Nagler <nagler@bivio.biz>
+To: Rob Nagler <nagler@bivio.biz>
+Subject: an image with an emdash in the filename
+Date: Sat, 21 Jan 2006 11:15:05 -0700
+
+
+--0__=0ABBF8F6DFC246528f9e8a93df938690918c0ABBF8F6DFC24652
+Content-Type: application/pdf; 
+        name="=?UTF-8?Q?2021_=E2=80=93_Foo_Bar=2Epdf?="
+Content-Disposition: attachment; 
+        filename="=?UTF-8?Q?2021_=E2=80=93_Foo_Bar=2Epdf?="
```

Comment:
Why did the underscore get changed to a space below?

Reply:
https://github.com/biviosoftware/perl-Bivio/blob/64-do-iterate-put-on-request/MIME/Word.pm#L26

Reply:
Thanks. (The link didn't work btw, not sure why.)

Reply:
Because I'm working on a different branch that hasn't been pushed and I used an emacs package to generate the link (`git-link`).